python/src/TaylorExpansionMoments_doc.i.in

```diff
@@ -12,87 +11,13 @@ limitStateVariable : :class:`~openturns.RandomVector`
 Notes
 -----
 Assuming that :math:`\uX` has its two first moments finite and that :math:`h` is sufficiently regular,
-the Taylor expansion method is used to approximate the mean and variance of a random variable :math:`\uY = h(\uX)`
-with respect to the mean and variance of the input random vector :math:`\uX`.
+the Taylor expansion method is used to approximate the mean and variance of the random variable :math:`\uY = h(\uX)`.
 
-We note:
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations:
 
-- :math:`\mu_X = \Expect{\uX}` the mean of the random vector :math:`\uX`
+- The one order  expansion of :math:`h` yields to an approximation of the mean and the variance of :math:`\uY`.
+- The second order  expansion of :math:`h` yields to an approximation of the mean :math:`\uY`.
```

Comment:
```suggestion
- the order one  expansion of :math:`h` yields to an approximation of the mean and the variance of :math:`\uY` ;
- the order two  expansion of :math:`h` yields to an approximation of the mean :math:`\uY`.
```

Reply:
I prefered the first-order and the second-order...

Reply:
That is ok for me. Your suggestion is better.